src/api/custom-schema/type-defs.ts

```diff
@@ -205,6 +205,13 @@ const typeDefs = `
     referenceNumber: Int
   }
 
+  type CreateAnAbandonedApplicationResponse {
+    success: Boolean!
+    id: String
+    referenceNumber: Int
+    status: String
```

Comment:
Is there a reason for having `status` here? As we don't have this for `CreateAnApplicationResponse`

Reply:
had it there for testing.  Removed